src/utils/add-corpus-storage-accounts/add-corpus-storage-account.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import json
+import uuid
+
+from azure.common.client_factory import get_client_from_cli_profile
+from azure.mgmt.eventgrid import EventGridManagementClient
+from azure.mgmt.eventgrid.models import EventSubscription
+from azure.mgmt.storage import StorageManagementClient
+from azure.mgmt.storage.models import (
+    AccessTier,
+    Kind,
+    Sku,
+    SkuName,
+    StorageAccountCreateParameters,
+)
+
+# This was generated randomly and should be preserved moving forwards
+STORAGE_GUID_NAMESPACE = uuid.UUID("f7eb528c-d849-4b81-9046-e7036f6203df")
+
+
+def get_base_event(
+    client: EventGridManagementClient, resource_group: str, location: str
+) -> EventSubscription:
+    for entry in client.event_subscriptions.list_regional_by_resource_group(
+        resource_group, location
+    ):
+        if (
+            entry.name == "onefuzz1"
```

Comment:
Where does this (`"onefuzz1"`) come from?

Reply:
(Is this the expected name for the primary account? If so, can we document that, and/or express it via a named constant?)